ci/glci/alicloud.py

```diff
@@ -273,3 +283,24 @@ def copy_image(self, src_image_id: str, dest_region: str) -> str:
 
 def parse_response(response):
     return json.loads(response)
+
+def main():
```

Comment:
I suppose this main function is debugging/testing code? not sure whether this should stay in